draft-ietf-quic-transport.md

```diff
@@ -4017,6 +4017,55 @@ transport to cancel a stream in response to receipt of a STOP_SENDING frame.
 
 # Security Considerations
 
+## Handshake Denial of Service
+
+As an encrypted and authenticated transport QUIC provides a range of protections
+against denial of service.  Once the cryptographic handshake is complete, QUIC
+endpoints discard packets that are not authenticated, greatly limiting the
```

Comment:
how about "most packets", since there clearly are unauthenticated packets that are accepted.

Reply:
Sure.  I was hoping that we could ultimately authenticate even those packets that don't have packet protection in some way.  For instance, by ensuring that the ICMP unreachable includes bytes from a packet we sent.  But maybe that's stretching the definition too far.